hydrolib/core/io/net/models.py

```diff
@@ -723,7 +723,7 @@ def _node_position(
         Returns:
             Union[np.int32, None]: The index of the coordinate. None if not found
         """
-        pos = np.where(np.isclose(arrx, x) & np.isclose(arry, y))[0]
+        pos = np.where(np.isclose(arrx, x, rtol=0.0) & np.isclose(arry, y, rtol=0.0))[0]
```

Comment:
~~Do you intentionally set relative tol to zero and keep the absolute tolerance to the default (1e-8)? 
What was going wrong until now?~~ 
Never mind, issue #298 explained it all.

Reply:
Indeed I meant "coordinate reference system". :)